`TestRootUTMPEntryExists` flakiness

## Failure

#### Link(s) to logs

- https://github.com/gravitational/teleport/actions/runs/12141893578/job/33855069443

#### Relevant snippet


```
panic: Assertion failed

goroutine 11438 [running]:
github.com/stretchr/testify/assert.(*CollectT).FailNow(0x17de2da0?)
	/go/pkg/mod/github.com/stretchr/testify@v1.9.0/assert/assertions.go:1924 +0x2b
github.com/stretchr/testify/require.NoError({0x17e3b898, 0xc00576e420}, {0x17de5040, 0xc005910d20}, {0x0, 0x0, 0x0})
	/go/pkg/mod/github.com/stretchr/testify@v1.9.0/require/require.go:1357 +0xe4
github.com/gravitational/teleport/integration.TestRootUTMPEntryExists.func1.2(0xc00576e420)
	/__w/teleport/teleport/integration/utmp_integration_test.go:121 +0xad
github.com/stretchr/testify/assert.EventuallyWithT.func1()
	/go/pkg/mod/github.com/stretchr/testify@v1.9.0/assert/assertions.go:1983 +0xb8
created by github.com/stretchr/testify/assert.EventuallyWithT in goroutine 11411
	/go/pkg/mod/github.com/stretchr/testify@v1.9.0/assert/assertions.go:1978 +0x42b
```

--- a/integration/utmp_integration_test.go
+++ b/integration/utmp_integration_test.go
@@ -118,10 +118,10 @@ func TestRootUTMPEntryExists(t *testing.T) {
 		require.NoError(t, err)
 
 		require.EventuallyWithTf(t, func(collect *assert.CollectT) {
-			require.NoError(collect, uacc.UserWithPtyInDatabase(s.utmpPath, teleportTestUser))
-			require.NoError(collect, uacc.UserWithPtyInDatabase(s.wtmpPath, teleportTestUser))
+			assert.NoError(collect, uacc.UserWithPtyInDatabase(s.utmpPath, teleportTestUser))
+			assert.NoError(collect, uacc.UserWithPtyInDatabase(s.wtmpPath, teleportTestUser))
 			// Ensure than an entry was not written to btmp.
-			require.True(collect, trace.IsNotFound(uacc.UserWithPtyInDatabase(s.btmpPath, teleportTestUser)), "unexpected error: %v", err)
+			assert.True(collect, trace.IsNotFound(uacc.UserWithPtyInDatabase(s.btmpPath, teleportTestUser)), "unexpected error: %v", err)
 		}, 5*time.Minute, time.Second, "did not detect utmp entry within 5 minutes")
 	})
 
@@ -154,10 +154,10 @@ func TestRootUTMPEntryExists(t *testing.T) {
 		require.NoError(t, err)
 
 		require.EventuallyWithT(t, func(collect *assert.CollectT) {
-			require.NoError(collect, uacc.UserWithPtyInDatabase(s.btmpPath, teleportFakeUser))
+			assert.NoError(collect, uacc.UserWithPtyInDatabase(s.btmpPath, teleportFakeUser))
 			// Ensure that entries were not written to utmp and wtmp
-			require.True(collect, trace.IsNotFound(uacc.UserWithPtyInDatabase(s.utmpPath, teleportFakeUser)), "unexpected error: %v", err)
-			require.True(collect, trace.IsNotFound(uacc.UserWithPtyInDatabase(s.wtmpPath, teleportFakeUser)), "unexpected error: %v", err)
+			assert.True(collect, trace.IsNotFound(uacc.UserWithPtyInDatabase(s.utmpPath, teleportFakeUser)), "unexpected error: %v", err)
+			assert.True(collect, trace.IsNotFound(uacc.UserWithPtyInDatabase(s.wtmpPath, teleportFakeUser)), "unexpected error: %v", err)
 		}, 5*time.Minute, time.Second, "did not detect btmp entry within 5 minutes")
 	})
 

